tests/e2e/helpers/global-setup.js

```diff
@@ -31,6 +31,9 @@ exports.getDocument = () => {
 		jsdom.JSDOM.fromURL(url, { resources: "usable", runScripts: "dangerously" }).then((dom) => {
 			dom.window.name = "jsdom";
 			global.window = dom.window;
+			global.navigator = {
```

Comment:
do you have a link where you found infos with regrds to this?

Reply:
https://www.appsloveworld.com/reactjs/100/37/mocha-react-navigator-is-not-defined

Reply:
thx, could you add it as a comment so we have it in case?

Reply:
done.